src/components/Popups/PopupReply.tsx

```diff
@@ -26,6 +27,12 @@ export const options: Partial<Popper.Options> = {
                 padding: 10,
             },
         },
+        {
+            name: 'eventListeners',
+            options: {
+                scroll: false,
```

Comment:
Could you explain the reasoning for adding `scroll: false`?

Reply:
Similar to the `PopupHighlight`, we add this so that PopperJS doesn't add it's own scroll event listener -- as a result, the popover just scrolls with the preview content